examples/feature_convert/cadquery/cube.py

```
@@ -0,0 +1,22 @@
+# OpenVMP, 2023
+#
+# Author: Roman Kuzmenko
+# Created: 2023-08-19
+#
+# Licensed under Apache License, Version 2.0.
+#
+
+import cadquery as cq
+
+if __name__ != "__cqgi__":
+    from cq_server.ui import ui, show_object
```

Ruff (0.8.2)

`cq_server.ui.ui` imported but unused

Remove unused import: `cq_server.ui.ui`

(F401)